pubspec.yaml

```
@@ -1,19 +1,19 @@
 name: observer_provider
 description: A Provider using Observable inputs
-version: 0.0.1+2
+version: 0.0.2-null-safety
```

```suggestion
version: 0.0.2-nullsafety.0
```

Follow version convention.